main.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const headers = ['Id', 'Task', 'Responsible', 'Status'];
+let listOfTasks;
+let listOfUsers; 
+
+getXMLHttpRequest('https://jsonplaceholder.typicode.com/todos', (response) => {
+  listOfTasks = response;
+  getXMLHttpRequest('https://jsonplaceholder.typicode.com/users', (response) => {

```

Comment:
Before sending the second request, you are waiting for first response, send requests in parallel